src/python/model/model.py

```
@@ -105,7 +105,7 @@ def add_file(self, file: ModelFile):
         :return:
         """
         key = Model.file_key(file)
-        self.logger.debug("LftpModel: Adding file '{}'".format(key))
+        self.logger.debug("LftpModel: Adding file '{}'".format(file.name))
```

Ruff (0.15.4)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)